apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19482,5 +19479,46 @@
+<e>       <p><l>vaqt<s n="n"/></l>                      <r>waqıt<s n="n"/></r></p></e>
+<e>       <p><l>Asal<s n="np"/><s n="ant"/><s n="f"/></l>   <r>Asal<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>uy<s n="n"/></l>                        <r>úy<s n="n"/></r></p></e>
+<e>       <p><l>ol<s n="v"/><s n="tv"/></l>             <r>al<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>tashqari<s n="n"/></l>                  <r>sırt<s n="n"/></r></p></e>
+<e>       <p><l>Oygul<s n="np"/><s n="ant"/><s n="f"/></l> <r>Aygúl<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>Baxtiyor<s n="np"/><s n="ant"/><s n="m"/></l> <r>Baxtıyar<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>ular<s n="prn"/><s n="pers"/></l>       <r>olar<s n="prn"/><s n="pers"/></r></p></e>
+<e>       <p><l>ulkan<s n="adj"/></l>                    <r>úlken<s n="adj"/></r></p></e>
+<e>       <p><l>qachon<s n="adv"/></l>                   <r>qashan<s n="adv"/></r></p></e>
+<e>       <p><l>xursand<s n="adj"/></l>                    <r>quwanıshli<s n="adj"/></r></p></e>
+<e>       <p><l>ob-havo<s n="n"/></l>                  <r>hawa rayı<s n="n"/></r></p></e>
+<e>       <p><l>yaqinlash<s n="v"/><s n="iv"/></l>             <r>jaqınlas<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>bekit<s n="v"/><s n="tv"/></l>             <r>jasır<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>ber<s n="v"/><s n="tv"/></l>             <r>ber<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>yoki<s n="cnjcoo"/></l>                  <r>yamasa<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>yosh<s n="n"/></l>                  <r>jas<s n="n"/></r></p></e>
+<e>       <p><l>sen<s n="prn"/><s n="pers"/></l>       <r>sen<s n="prn"/><s n="pers"/></r></p></e>
+<e>       <p><l>yer<s n="n"/></l>                  <r>jer<s n="n"/></r></p></e>
+<e>       <p><l>topish<s n="v"/><s n="iv"/></l>             <r>tabıw<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>kir<s n="v"/><s n="iv"/></l>             <r>kir<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>kirish<s n="v"/><s n="iv"/></l>             <r>kiriw<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>bolakay<s n="n"/></l>                     <r>bala<s n="n"/></r></p></e>
+<e>       <p><l>haliyam<s n="adv"/></l>                   <r>ele<s n="adv"/></r></p></e>
+<e>       <p><l>qanday<s n="adv"/></l>                   <r>qanday<s n="adv"/></r></p></e>
+<e>       <p><l>singli<s n="n"/></l>                     <r>qarındas<s n="n"/></r></p></e>
+<e>       <p><l>gapir<s n="v"/><s n="tv"/></l>             <r>sóyle<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>boshqa<s n="adj"/></l>                   <r>basqa<s n="adj"/></r></p></e>
+<e>       <p><l>tag<s n="n"/></l>                     <r>ast<s n="n"/></r></p></e>
+<e>       <p><l>yon<s n="n"/></l>                     <r>qaptal<s n="n"/></r></p></e>
+<e>       <p><l>soʻng<s n="adv"/></l>                   <r>soń<s n="adv"/></r></p></e>
```

Comment:
I think this should be "post", unless you have a reason to categorise it as "adv" instead?

Specifically, you can use keyin either with a noun (post) or on its own (adv), right?  Like "keyin men úyge kirdim" (adv) or "mektepten keyin men úyge kirdim" (post), but with soń, you can't say ??"són men úyge kirdim", right?